test/functional/buildAndPackage/src/net/adoptopenjdk/test/VendorPropertiesTest.java

```diff
@@ -89,87 +114,94 @@ public void vmPropertiesPointToVendor() {
          */
         boolean supports(String vendor);
 
         /**
          * Checks whether the output of {@code java -version} is acceptable.
+         * @param value the value to be validated.
          */
         void javaVersion(String value);
 
         /**
          * Checks the value of {@code java.vendor}.
+         * @param value the value to be validated.
          */
         void javaVendor(String value);
 
         /**
          * Checks the value of {@code java.vendor.url}.
+         * @param value the value to be validated.
          */
         void javaVendorUrl(String value);
 
         /**
          * Checks the value of {@code java.vendor.url.bug}.
+         * @param value the value to be validated.
          */
         void javaVendorUrlBug(String value);
 
         /**
          * Checks the value of {@code java.vendor.version}.
+         * @param value the value to be validated.
          */
         void javaVendorVersion(String value);
 
         /**
          * Checks the value of {@code java.vm.vendor}.
+         * @param value the value to be validated.
          */
         void javaVmVendor(String value);
 
         /**
          * Checks the value of {@code java.vm.version}.
+         * @param value the value to be validated.
          */
         void javaVmVersion(String value);
     }
 
     private static class AdoptOpenJDKPropertiesChecks implements VmPropertiesChecks {
 
         @Override
-        public boolean supports(String vendor) {
+        public boolean supports(final String vendor) {
             return vendor.toLowerCase(Locale.US).equals("adoptopenjdk");
         }
 
         @Override
-        public void javaVersion(String value) {
+        public void javaVersion(final String value) {
             assertTrue(value.contains("AdoptOpenJDK"));
         }
 
         @Override
-        public void javaVendor(String value) {
+        public void javaVendor(final String value) {
             assertEquals(value, "AdoptOpenJDK");
         }
 
         @Override
-        public void javaVendorUrl(String value) {
+        public void javaVendorUrl(final String value) {
             assertEquals(value, "https://adoptium.net/");
         }
 
         @Override
-        public void javaVendorUrlBug(String value) {
+        public void javaVendorUrlBug(final String value) {
             assertEquals(value, "https://github.com/AdoptOpenJDK/openjdk-support/issues");
         }
 
         @Override
-        public void javaVendorVersion(String value) {
+        public void javaVendorVersion(final String value) {
             assertNotEquals(value.replaceAll("[^0-9]", "").length(), 0,
                     "java.vendor.version contains no numbers: " + value);
         }
 
         @Override
-        public void javaVmVendor(String value) {
+        public void javaVmVendor(final String value) {
             assertTrue(value.equals("AdoptOpenJDK") || value.equals("Eclipse OpenJ9"));
```

Comment:
Line 191 of `JdkVersion.java` is expecting `"Eclipse OpenJ9 VM"`.

Reply:
I think that's because JdkVersion.java line 191 is expecting the system property of java.vm.name, while this is testing the value of java.vm.name.